app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/SingleDisposalsTriageController.scala

```diff
@@ -413,6 +414,10 @@ class SingleDisposalsTriageController @Inject() (
     )
   }
 
+  def assetTypeNotYetImplemented(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
+    Ok(assetTypeNotYetImplementedPage())
```

Comment:
can we do at least a `withSingleDisposalTriageAnswers` check here? Just so we have a better idea that the user is supposed to see this page